lib/widgets/components/entry/register/register_continue_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:coffeecard/widgets/components/rounded_button.dart';
+import 'package:flutter/material.dart';
+
+class RegisterContinueButton extends StatefulWidget {
```

Comment:
I don't fully understand the need of the button being a stateful widget. Could you elaborate why this is necessary, or if it isn't make it stateless instead?

Reply:
its not, i forgot to remove it :) fixing